### ISSUE TITLE
Migrate the drag-and-drop reordering

### ISSUE THREAD

**Issue by alachaum** [KeypDemoCorp/awesome-ui#211]
This was requested by several customers during recent calls.

We have seen related errors in the monitoring dashboard recently.

Accessibility should be considered when implementing this.
